exec-unittests

```diff
@@ -186,18 +181,30 @@ for _major in ${major_list}; do
                 (( count++ ))
             fi
         done > ${_para_jobs_file}
-        parallel --will-cite -k --lb -a ${_para_jobs_file} ${para_jobs_arg} para_run_legacy
-        if [[ ${?} -ne 0 ]]; then
-            rc=1
-        fi
-        if [[ ${count} -eq 0 && "${subtst}" != "python" ]]; then
-            printf -- "Error - unrecognized sub-test, '%s'\n" "${subtst}" >&2
-            rc=1
+        if [[ -s ${_para_jobs_file} ]]; then
+			parallel --will-cite -k --lb -a ${_para_jobs_file} ${para_jobs_arg} para_run_legacy
```

Comment:
This file appears to use spaces for indentation, but line 185 (et al.) are using hard-tabs.  We should be consistent.